etl/steps/data/garden/energy/2024-10-30/photovoltaic_cost_and_capacity.meta.yml

```diff
@@ -0,0 +1,30 @@
+definitions:
+  common:
+    processing_level: major
+    presentation:
+      topic_tags:
+        - Energy
+
+dataset:
+  update_period_days: 365
+
+tables:
```

Comment:
It seems you can add short_unit to the metadata, here or in the script.

Reply:
There's no need, it's properly propagated (I just checked).